addon/templates/components/bootstrap-textarea.hbs

```diff
@@ -1,22 +1,10 @@
-{{#if hasBlock}}
-  {{yield inputId}}
-{{else}}
-  {{#if label}}
-    <label for="{{inputId}}" class="control-label {{if hasColumns labelColumnCss ''}} {{if srOnly 'sr-only' ''}}">{{label}}</label>
-  {{/if}}
-  {{#if hasColumns}}
-    <div class="{{inputColumnCss}}">
-      {{textarea
-        tabindex=tabindex
-        value=value
-        elementId=inputId
-        placeholder=placeholder
-        classNames="form-control"
-        rows=rows
-        cols=cols
-        disabled=disabled}}
-    </div>
+<div class="form-group bootstrap-input-component {{if hasSuccess 'has-success'}} {{if hasWarning 'has-warning'}} {{if hasError 'has-error'}} {{if required 'required'}}">
```

Comment:
Should this class name be something like `bootstrap-textarea-component` instead?